src/coreclr/jit/emit.h

```diff
@@ -576,6 +576,10 @@ class emitter
 #elif defined(TARGET_ARM64)
```

Comment:
`CodeGen` and `emitter` are late phases. They should be all replaced by the LLVM part. Is it possible to ifdef them out?

Reply:
Do you think I should be able #if out the Lowering class?  https://github.com/dotnet/runtimelab/blob/feature/NativeAOT/src/coreclr/jit/lower.h

Reply:
Have removed those + lsra + lower.  There's a few places that require `#ifdefs` to allow compilation to succeed.  In some places  where I'm not sure what's best I've left the wasm build throwing an `assert`.

Reply:
I think `inst*` can be remove too.

Reply:
Thanks, removed.